bigtable/google/cloud/bigtable/table.py

```diff
@@ -373,3 +411,65 @@ def _create_row_request(table_name, row_key=None, start_key=None, end_key=None,
         message.rows.row_ranges.add(**range_kwargs)
 
     return message
+
+
+def _mutate_rows_request(table_name, rows):
+    """Creates a request to mutate rows in a table.
+
+    :type table_name: str
+    :param table_name: The name of the table to write to.
+
+    :type rows: list
+    :param rows: List or other iterable of :class:`.DirectRow` instances.
+
+    :rtype: :class:`data_messages_v2_pb2.MutateRowsRequest`
+    :returns: The ``MutateRowsRequest`` protobuf corresponding to the inputs.
+    :raises: :exc:`~.table.TooManyMutationsError` if the number of mutations is
+             greater than 100,000
+    """
+    request_pb = data_messages_v2_pb2.MutateRowsRequest(table_name=table_name)
+    mutations_count = 0
+    for row in rows:
+        _check_row_table_name(table_name, row)
+        _check_row_type(row)
+        entry = request_pb.entries.add()
+        entry.row_key = row.row_key
+        for mutation in row._get_mutations(None):
+            mutations_count += 1
+            entry.mutations.add().CopyFrom(mutation)
+    if mutations_count > _MAX_BULK_MUTATIONS:
+        raise TooManyMutationsError('Maximum number of mutations is %s' %
+                                    _MAX_BULK_MUTATIONS)
```

Comment:
Always pass a tuple to a format string. So it'd be

```python
    if mutations_count > _MAX_BULK_MUTATIONS:
        raise TooManyMutationsError(
            'Maximum number of mutations is %s' % (_MAX_BULK_MUTATIONS,))
```

or alternatively you could just forego the format string

```python
    if mutations_count > _MAX_BULK_MUTATIONS:
        raise TooManyMutationsError(
            'Maximum number of mutations', _MAX_BULK_MUTATIONS)
```